src/bin/tikv-ctl.rs

```diff
@@ -1190,9 +1202,18 @@ fn main() {
         debug_executor.set_region_tombstone_after_remove_peer(mgr, &cfg, regions);
     } else if let Some(matches) = matches.subcommand_matches("unsafe-recover") {
         if let Some(matches) = matches.subcommand_matches("remove-fail-stores") {
+            let region_ids = matches
+                .values_of("regions")
+                .map(|ids| {
+                    ids.map(|r| r.parse())
+                        .collect::<Result<Vec<_>, _>>()
+                        .expect("parse regions fail")
+                })
+                .unwrap_or_default();
```

Comment:
If you choose to change the function signature of `remove_failed_stores` to take an `Option<Vec<u64>>` for `region_ids` then this unwrap can go away! Yay!

Reply:
Great! :)